src/labbook/components/thumbnail-chooser/thumbnail-chooser.tsx

```diff
@@ -14,7 +14,7 @@ interface basicButtonProps {
 
 const PrevButton:React.FC<basicButtonProps> = (props: basicButtonProps) => {
   const {onClick, enabled} = props;
-  const backClasses = classNames(css["button-back"], {[css.disabled]: !enabled});
+  const backClasses = classNames(css["arrow-button"], css["back"], {[css.disabled]: !enabled});
```

Comment:
I didn't see "back" or "forward" classes in `./thumbnail-chooser.scss` -- Not sure if the class name is used in tests or somewhere else, but maybe we can remove this.

I think originally the name `button-back` came from the Zeplin specs. This might be a naming convention Michael uses when talking about the background div in which an SVG button sits.  I see here it's use was confusing because of the forward and backward navigation.

Reply:
You're right, I was messing around with adding those classnames then forgot to delete them. I just removed them & committed.

In terms of the screenshots above, I made the area with the snapshots horizontally scrollable, so even though it looks like you can't see all of them, you can actually. Maybe that's not clear enough, though.